### PR TITLE
Charter changes proposal 2020

### PR DESCRIPTION
**Pull request by phalt**
I have taken the proposed changes from our internal document here: https://docs.google.com/document/d/1Uu_n7xIjFSCHED0WEVGj5y8Hje_D825SoA4QB7gx1Sk/edit#

Most of it was simple, but Rule 6 confused me somewhat, a review would be great.

TODO:

- [x] Finalise changes in this PR
- [ ] Improve this text description for wider audience
- [x] Ensure limited Issue/Pull Request creation on this repository
- [x] Publish this publicly
- [ ] Tag the changed version if this is merged (versioning recommendation is the merge date i.e. 20-09-2020)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,29 +29,52 @@ The Charity aims to organise an annual conference ("Kiwi PyCon"). This means:
...
+The Charity may designate certain benefits, areas, events, or other resources as Members Benefits, and restrict access to these for the benefit of current members.
 
-Membership of the NZPUG mailing list and membership of the New Zealand Python User Group Incorporated are independent.
+Subscription, membership, or presence in any area operated by the New Zealand Python User Group does not imply membership of the Charity itself.
 
 ## Rule 4
+
+### Rule 4a
+
 A member can choose to leave the Charity at any time, without refund of their fees. The member wishing to leave must submit written notice (email or letter) to the committee to this effect. Non-payment of subscription fees will result in termination of membership and all associated benefits.
```
Comment (rhyven) [inline README.md:54]: Remove leading space

Comment (rhyven) [reply README.md:54]: ... Actually, that might just be GitHub

File: README.md [hunk 2]
```diff
@@ -25,29 +29,52 @@ The Charity aims to organise an annual conference ("Kiwi PyCon"). This means:
...
+Any committee member who is found to be in breach of the Code of Conduct will immediately cease their office, and will be ineligible to sit on the committee for a period of two years, to be counted from the following AGM.  A by-election will be held to fill the vacated position, unless the AGM is due to occur within two calendar months.
+
+The committee may review any member’s membership if they are prosecuted for a crime.
+
 "Serious misconduct" is held at the discretion of the committee of the Charity, and any grievances should be brought before the committee.
 
 ## Rule 5
-The Rules of the Charity may be altered, added to or rescinded at an annual general meeting (AGM) or, in special circumstances, at a special meeting held for the purpose. Such meetings must be announced ahead of time and are open to all members of the Charity. Only members of the Charity can vote.
+
+The Rules of the Charity may be altered, added to or rescinded at an annual general meeting (AGM) or, in special circumstances, at a special meeting held for the purpose. These meetings will be conduted in accordance with Rule 6. Only members of the Charity can vote.
```
Comment (rhyven) [inline README.md]: Ooops, 'conduted' -> 'conducted'.

File: README.md [hunk 3]
```diff
@@ -138,11 +140,13 @@ The standard term of appointment is one year, or until the next AGM, whichever i
 All officers must qualify under section 16 of the Charities Act, and corresponding forms must be signed to certify this.
 
 ## Rule 8
+
 The common seal of the Charity shall be held with the secretary.
 
-The seal can be used to verify documents on behalf of the Charity, either by the committee or officers of the committee at an AGM or special meeting, or by the president of the Charity if s/he deems it appropriate.
+The seal can be used to verify documents on behalf of the Charity, either by the committee or officers of the committee at an AGM or special meeting, or by the president of the Charity if they deems it appropriate.
```
Comment (rhyven) [inline README.md]: "they deems" -> "they deem"

File: README.md [hunk 4]
```diff
@@ -58,76 +85,51 @@ No addition to or alteration of the non-profit aims, personal benefit clause or
...
-The purposes of a special meeting is to conduct any urgent Charity business, including but not limited to:
+At a General Meeting, quorum shall be 50% of the Charity membership, or 15 people, whichever number is smaller.
 
-* conducting any urgent financial activity which cannot wait until the next general meeting
-* disciplinary action to be taken against a Charity member
-* changes to the charter or Rules of the Charity
+### Rule 6b
 
-The committee can call a special general meeting by the request of any committee member. A member may call a SGM if backed by at least 10% of the Charity's membership.
+Committee Meetings
```
Comment (rhyven) [inline README.md]: Make this a ### header

**Review by rhyven — COMMENTED** [review]
Hey Paul, this is all looking great to me!  Have made a couple comments where there are small edits to be made

**Comment by phalt** [comment]
@rhyven Hey Eric, thanks for the review. Changes made!

File: README.md [hunk 5]
```diff
@@ -58,76 +85,52 @@ No addition to or alteration of the non-profit aims, personal benefit clause or
...
+An SGM can be called to conduct any urgent Charity business that cannot wait until the AGM, and is of such significance that it requires the involvement of the all the members, including but not limited to:
 
-### Regular Meetings
-Regular meetings are to be held at a monthly interval. Announcement of meetings and the topic of the meeting will be made on the Charity mailing list at suitable intervals prior to the event, such as one week, two days, one day etc.
+* conducting any financial activity which cannot wait until the next general meeting.
+* disciplinary action to be taken against a Charity member.
+* changes to the charter or rules of the Charity.
+* any other matters requiring the attention of all members of the society which cannot wait until the next AGM.
+* An SGM can be called at the request of any committee member.  Any member of the Charity may call an SGM if backed by at least 10% of the Charity's membership.
+* Any decisions to be made at a General Meeting will be made by vote.  The method of voting will be defined in the announcement of the General Meeting.  Only paid members in the current financial year are eligible to vote.
```
Comment (rhyven) [inline README.md:115]: Move "any decisions to be made at a General Meeting" line.  Line 115 -> line 98.  Remove bullet-point.

File: README.md [hunk 6]
```diff
@@ -58,76 +85,52 @@ No addition to or alteration of the non-profit aims, personal benefit clause or
...
 
-Regular meetings shall conduct of such content as is discussed by the Charity, including but not limited to:
+A simple majority will indicate the success of any motion put to vote, except in the cases specified in Rule 5. The president of the Charity (or the vice president in their absence) has a casting vote.  In the event of a draw, the motion must be deferred to a following General Meeting.
```
Comment (rhyven) [inline README.md:117]: Move "A simple majority" clause to the overall "General Meetings" opening stanza.  Line 117 -> line 98.

File: README.md [hunk 7]
```diff
@@ -58,76 +85,52 @@ No addition to or alteration of the non-profit aims, personal benefit clause or
...
 
-* presentations on a suitable topic, preferably one which fits within the purposes and aims of the Charity
-* "hands on" technical assistance for all attendees. Such meetings are designed to help members and non-members alike with software written in the Python programming language.
-* other topics considered suitable by the Charity members, as indicated by a simple consensus.
-* Members and non-members alike can give or organise presentations or discussions.
+At a General Meeting, quorum shall be 50% of the Charity membership, or 15 people, whichever number is smaller.
```
Comment (rhyven) [inline README.md:119]: Move "At a General Meeting, quoum" line to "General meetings" opening stanza.

Line 119 -> Line 98

File: README.md [hunk 8]
```diff
@@ -58,76 +85,52 @@ No addition to or alteration of the non-profit aims, personal benefit clause or
...
 
-Throughout New Zealand, monthly meetings may be held in multiple locations.
+Any meeting - including the AGM - may be conducted either in-person or online, and either synchronously or asynchronously, or any combination thereof.  The venue or method selected for the meeting must be clearly expressed beforehand and in all invitations.
 
-### Annual General Meetings
-Annual General meetings are to be held in the month of September.
+### Rule 6a
 
-Annual General Meetings shall be announced on the Charity's electronic mailing list a month before they occur, and at the previous months meeting. A reminder shall be posted to the mailing list at suitable intervals prior to the event, such as two weeks, one week, two days, one day, etc.
+General Meetings
```
Comment (rhyven) [inline README.md:93]: Make "General Meetings" a section header - ###

**Review by rhyven — APPROVED** [review]

**Pull request merged**